css/login-page.css

```diff
@@ -0,0 +1,69 @@
+body{
+    background-color: gray;
+    background-image: url("background-soft4.jpeg");
+    background-repeat: no-repeat;
+    overflow: hidden;
+}
+
+div#forma{
+    width: 300px;
+    margin: 0 auto;
+    padding-top: 40px;
+    display: block;
+    text-align: center;
+}
+
+h1{
+    text-align: center;
+    color: white;
+}
+
+div.jumbotron-fluid{
```

Comment:
Think about a new person joining the codbase.
Could he understand what this class does without reading it? (Spoiler no)

Use better naming

Reply:
yep, but that's a class that work whit the bootstrap library so i cant change it

Reply:
80% positive that you can avoid using it and find a pure CSS way to achieve the same thing.
In this project you can only use CSS, js and HTML.
Then you'll be able to use SCSS, ts and only later frameworks.